src/course-home/mentoring-tab/messages.ts

```diff
@@ -0,0 +1,42 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
+
+const messages = defineMessages({
+  title: {
+    id: 'learning.mentoring.title',
+    defaultMessage: 'Mentoring Assistant',
```

Comment:
[optional]: Similar to `Your progress` (first letter is capital, rest are lowercase)

```suggestion
    defaultMessage: 'Mentoring assistant',
```